content/projects/java-specific/threads/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+content_type: project
+flavours:
+- java
+prerequisites:
+  hard:
+  - topics/java-specific/concurrency
+  - topics/java-specific/functional-interface
+  soft:
+  - projects/java-specific/functional-interface
+ready: true
+submission_type: repo
+title: Java Threads
+---
+
+### The Chicken and the Farmer
+
+Have you read the book Animal Farm well its not so important for this project but if you havent read it you should; anyways, in this project you are going to create a Chicken and a Farmer each of these will be there own functions and these are the rules
+
+1. Have the Chicken and the Farmer run on different threads
+
+2. The Chicken should put eggs in a Basket ***(static array/list of size 1)*** and the Farmer should take eggs from the basket
+
+3. The Farmer cannot take from an empty basket
+
+4. The Chicken cannot add to a full basket
+
+5. The basket is full if it has 1 egg
+
+
+That is it, pretty simple. have fun
```

Comment:
If you call something simple and people struggle then they sometimes get quite discouraged.

Maybe say something like:

```
We kept the instructions open-ended on purpose. This is meant to have a bit of room for interpretation, there is no one perfect solution to this. We are very interested to see how you structure your solution.

Oh yeah, and remember to test your work. Always.
```